app/cms/utils/fetch-sporks.tsx

```diff
@@ -0,0 +1,63 @@
+import { Octokit } from "@octokit/rest"
+import { cachified } from "../cache.server"
+import { redisCache } from "../redis.server"
+import { SporksCardProps } from "~/ui/design-system/src/lib/Components/SporksCard"
+
+export const fetchSporks = async () => {
+  return await cachified({
+    cache: redisCache,
+    maxAge: 1000 * 60 * 60 * 4,
+    forceFresh: process.env.FORCE_REFRESH === "true",
+    key: `sporks`,
+    getFreshValue: fetchFreshSporks,
+  })
+}
+
+export const fetchFreshSporks = async () => {
+  const octokit = new Octokit({
```

Comment:
Should we just import the `octokit` instance from `app/cms/github.server.ts` instead? That way we get the throttling options and any logging pre-configured.

Reply:
That's a good call! I'll swap it out.

Reply:
@joekrill updated!